ticdc/ticdc-open-api-v2.md

```diff
@@ -321,17 +321,18 @@ The `mounter` parameter is described as follows:
-| `transaction_atomicity` | `STRING` type. The atomicity level of the transaction. (Optional)  |
+| Parameter name | Description                                                                                                                                                                                                    |
+|:-----------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `column_selectors`      | The column selector configuration. (Optional)                                                                                                                                                                  |
+| `csv`                   | The CSV configuration. (Optional)                                                                                                                                                                              |
+| `date_separator`        | `STRING` type. Indicates the date separator type of the file directory. Value options are `none`, `year`, `month`, and `day`. `none` is the default value and means that the date is not separated. (Optional) |
+| `dispatchers`           | An configuration array for event dispatching. (Optional)                                                                                                                                                       |
+| `encoder_concurrency`   | `INT` type. The number of encoder threads in the MQ sink. The default value is `16`. (Optional)                                                                                                                |
+| `protocol`              | `STRING` type. For MQ sinks, you can specify the protocol format of the message. The following protocols are currently supported: `canal-json`, `open-protocol`, `canal`, `avro`, and `maxwell`.               |
+| `schema_registry`       | `STRING` type. The schema registry address. (Optional)                                                                                                                                                         |
+| `terminator`            | `STRING` type. The terminator is used to separate two data change events. The default value is null, which means `"\r\n"` is used as the terminator. (Optional)                                                |
+| `transaction_atomicity` | `STRING` type. The atomicity level of the transaction. (Optional)                                                                                                                                              |
+| `only_output_updated_columns` | `BOOLEAN` type. For MQ sinks using the `canal-json` or `open-protocol` protocol, you can specify whether only output the modified columns. The default value is `false`. (Optional)  |
```

Comment:
```suggestion
| `only_output_updated_columns` | `BOOLEAN` type. For MQ sinks using the `canal-json` or `open-protocol` protocol, you can specify whether only output the modified columns. The default value is `false`.  |
```

Reply:
(Optional) will be added in another PR https://github.com/pingcap/docs/pull/15046